gui/browsable/src/RFieldHolder.hxx

```diff
@@ -37,7 +37,7 @@ public:
    {
    }
 
-   const TClass *GetClass() const override { return TClass::GetClass<ROOT::Experimental::RNTuple>(); }
+   const TClass *GetClass() const override { return TClass::GetClass<ROOT::RNTuple>(); }
```

Comment:
Not strictly needed (and maybe annoying to do at this point) but I would recommend to have the commit that move all the classes out of the `Experimental` namespace to be the very last commit in the PR (so that technically there is no 'normal' way to get a `RNTuple` file with non-experiemental names and a not 'completely' 1.0 format.